.gitignore

```
@@ -11,3 +11,7 @@
 /build/
 /.classpath
 /.project
+/target/
+/project/target/
+/.bsp/
+/project/project/target/
```

you can change the above with the following, and it should be equivalent (and more future-proof)

```suggestion
target/
/.bsp/
```

---

Thankyou for the suggestion, I have made this change.